src/sentry/api/endpoints/project_rule_actions.py

```diff
@@ -62,8 +63,13 @@ def post(self, request: Request, project) -> Response:
         )
         rule = Rule(id=-1, project=project, data=data)
 
+        sample_event_fingerprint = f"sample_evt:{str(timezone.now())}"
```

Comment:
did we want to give users the option of whether to create a new issue when they send a test notification? i forgot

Reply:
My initial impression was that it would be a more confusing user experience. It won't always be clear to a user what generating a new group will do for testing 😅